src/work-allocation/containers/my-tasks/my-tasks.component.spec.ts

```diff
@@ -173,4 +180,15 @@ describe('MyTasksComponent', () => {
     // Ensure the correct attempt has been made to navigate.
     expect(navigateSpy).toHaveBeenCalledWith([`/tasks/${task.id}/${actionId}/`], jasmine.any(Object));
   });
+
+  fit('should allow setting the release 2 details', () => {
```

Comment:
Cannot be `fit` or you're only running this test.

Reply:
Please remove f, again agree with Paul G